skl2onnx/operator_converters/calibrated_classifier_cv.py

```diff
@@ -9,7 +9,7 @@
 from ..proto import onnx_proto
 from ..common._apply_operation import (
     apply_abs, apply_add, apply_cast, apply_concat, apply_div, apply_exp,
-    apply_mul, apply_reshape, apply_sub)
+    apply_mul, apply_reshape, apply_sub, apply_clip)
```

Comment:
I would prefer the alphabetical order of imports to be maintained, improves readability.